stories/demos/components/Editors/TextEditor.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useRef, useLayoutEffect } from 'react';
+import { Editor2Props } from '../../../../src/types';
+
+export function TextEditor<R>({ value, onChange, rowHeight }: Editor2Props<string, R>) {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useLayoutEffect(() => {
```

Comment:
We no longer have the bug where the input value is initially selected and typing another key deletes it